.gitignore

```diff
@@ -32,3 +32,11 @@ playground.xcworkspace
 /.buckconfig.local
 /.buckd
 
+# CocoaPods
+*.xcworkspace
+Podfile.lock
```

Comment:
https://guides.cocoapods.org/using/using-cocoapods.html -> "Whether or not you check in the Pods directory, the Podfile and Podfile.lock should always be kept under version control."

Reply:
hi @wiseoldduck , thanks for pointing out this, I've updated the commit :)

btw, it seems the repo's example projects don't host `Podfile.lock` at the moment, do you think it is good to add them to be under version control?

Reply:
Yeah examples ideally should have `Podfile.lock`. Not a big deal I'd say.

Reply:
Just noticing the `build.sh`, which will be ran by CI, ignores `Podfile.lock` in the example projects every time it makes a new build

```
...
if [ -f "${example}/Podfile" ]; then
        echo "Using CocoaPods"
        if [ -f "${example}/Podfile.lock" ]; then
            rm "$example/Podfile.lock"
        fi
...
```

So I think it's safe to not commit those `Podfile.lock` at the moment :p